x/subspaces/types/hooks.go

```diff
@@ -17,13 +17,16 @@ type SubspacesHooks interface {
 	AfterSubspaceSaved(ctx sdk.Context, subspaceID uint64)   // Must be called when a subspace is saved
 	AfterSubspaceDeleted(ctx sdk.Context, subspaceID uint64) // Must be called when a subspace is deleted
 
+	AfterSubspaceSectionSaved(ctx sdk.Context, subspaceID uint64, sectionID uint32)   // Must be called when a subspace section is saved
+	AfterSubspaceSectionDeleted(ctx sdk.Context, subspaceID uint64, sectionID uint32) // Must be called when a subspace section is deleted
+
```

Comment:
I think we should we also add `AfterSubspaceSectionMoved`

Reply:
I don't think it's worth it. That would require to have a custom keeper method that deals with moving sections. Instead, the current implementation will simply call `AfterSubspaceSectionSaved` when a section is moved (since it's just about changing its `ParentID` value). The module using this hook will simply be able to check the previously stored data and react properly if they are interested in doing so.